src/shogun/modelselection/ParameterCombination.cpp

```diff
@@ -700,7 +698,7 @@ CParameterCombination* CParameterCombination::copy_tree() const
 
 void CParameterCombination::apply_to_machine(CMachine* machine) const
 {
-	apply_to_modsel_parameter(machine->m_model_selection_parameters);
```

Comment:
same here. The danger with just removing this line is that we forget to re-add it later when we have a replacement. So rather than changing behaviour of code, let's just put a runtime error